span-normalizer/span-normalizer-constants/src/main/java/org/hypertrace/core/semantic/convention/constants/messaging/OtelMessagingSemanticConventions.java

```diff
@@ -6,7 +6,7 @@
   MESSAGING_URL("messaging.url"),
   RABBITMQ_MESSAGING_SYSTEM_VALUE("rabbitmq"),
   RABBITMQ_ROUTING_KEY("messaging.rabbitmq.routing_key"),
-  RABIT_MQ_MESSAGING_SYSTEM_VALUE("rabbitmq"),
+  RABBITMQ_COMMAND("rabbitmq.command"),
```

Comment:
Don't see `rabbitmq.command` in otel messaging semantics - https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/trace/semantic_conventions/messaging.md. Is this from open tracing?

Reply:
@Aka-shi, this is coming from javaagent instrumentation as you can see here: https://github.com/open-telemetry/opentelemetry-java-instrumentation/blob/3a206db032502d93de2b12a93d6f1b4d0fb57fdb/instrumentation/rabbitmq-2.7/javaagent/src/main/java/io/opentelemetry/javaagent/instrumentation/rabbitmq/RabbitTracer.java